pyproject.toml

```diff
@@ -29,14 +32,76 @@ classifiers = [
     "Topic :: Utilities",
     "Typing :: Typed",
 ]
-dynamic = ["version", "dependencies", "optional-dependencies"]
+dynamic = ["version"]
 
 [project.urls]
 Changelog = "https://docs.disnake.dev/page/whats_new.html"
 Homepage = "https://github.com/DisnakeDev/disnake"
 Documentation = "https://docs.disnake.dev/"
 Repository = "https://github.com/DisnakeDev/disnake"
 
+[project.optional-dependencies]
+speed = [
+    "orjson~=3.6",
+    # taken from aiohttp[speedups]
+    "aiodns>=1.1",
+    "Brotli",
+    'cchardet; python_version < "3.10"',
+]
+voice = [
+    "PyNaCl>=1.3.0,<1.5",
+]
+docs = [
+    "docutils>=0.17",
+    "sphinx==5.3.0",
+    "sphinxcontrib-trio~=1.1.2",
+    "sphinx-hoverxref==1.3.0",
+    "sphinx-autobuild~=2021.3",
+    "sphinxcontrib-towncrier==0.3.0a0",
+    "towncrier==22.8.0",
+    "sphinx-notfound-page==0.8.3",
+]
+discord = ["discord-disnake"]
+
+[tool.pdm.dev-dependencies]
+tools =[
+    "pre-commit~=2.19.0",
+    "taskipy~=1.10.1",
+    "slotscheck~=0.15.0",
+    "python-dotenv~=0.20.0",
+    "towncrier==22.8.0",
+    # checks MANIFEST.in + sdist
+    "check-manifest==0.48",
+    "twine>=4.0.1",
+    "build>=0.9.0",
```

Comment:
There's no need to have these in the requirements, at least as long as there's no pypi release workflow.

Reply:
https://github.com/DisnakeDev/disnake/pull/836/commits/a49789dbb0c3d906fae4c11b6c890d034ff1f2d0